Bottom: 3ff6770d7c8a78981340d2d9970980ac0eb081d8
Top:    b2ad3f732b168d3d2088409b69976b3ada525124
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 20:27:35 +0100

Refresh of hid-gtk-refactor-to-remove-som

---

diff --git a/src/hid/gtk/ghid-layer-selector.c b/src/hid/gtk/ghid-layer-selector.c
index 248e2d3..7648261 100644
--- a/src/hid/gtk/ghid-layer-selector.c
+++ b/src/hid/gtk/ghid-layer-selector.c
@@ -515,15 +515,16 @@ ghid_layer_selector_add_layer (GHidLayerSelector *ls,
 
   new_layer = malloc (sizeof (*new_layer));
 
-  gtk_list_store_set (ls->list_store, &iter,
-                      STRUCT_COL, new_layer,
-                      USER_ID_COL, user_id,
-                      VISIBLE_COL, visible,
-                      COLOR_COL, color_string,
-                      TEXT_COL, name,
-                      FONT_COL, activatable ? NULL : "Italic",
+  gtk_list_store_set (ls->list_store,
+                      &iter,
+                      STRUCT_COL,      new_layer,
+                      USER_ID_COL,     user_id,
+                      VISIBLE_COL,     visible,
+                      COLOR_COL,       color_string,
+                      TEXT_COL,        name,
+                      FONT_COL,        activatable ? NULL : "Italic",
                       ACTIVATABLE_COL, activatable,
-                      SEPARATOR_COL, FALSE,
+                      SEPARATOR_COL,   FALSE,
                       -1);
 
   /* -- Setup new actions -- */
